modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/NetUtils.java

```diff
@@ -22,37 +25,31 @@ public class NetUtils {
 
     private NetUtils() {}
 
+    // Accessors to the extended socket options reduce the proliferation of the non-portable
+    // ExtendedSocketOptions type.
+
     /**
-     * Returns the extended TCP_KEEPIDLE socket option, if available on this JDK
+     * Returns the extended TCP_KEEPIDLE socket option.
      */
-    public static SocketOption<Integer> getTcpKeepIdleSocketOptionOrNull() {
-        return getExtendedSocketOptionOrNull("TCP_KEEPIDLE");
+    @SuppressForbidden(reason = "access to non-portable socket option requied")
```

Comment:
nit:

```suggestion
    @SuppressForbidden(reason = "access to non-portable socket option required")
```

(in all the `SuppressForbidden` lines)

Reply:
Done.